src/helpers/mapProps.ts

```diff
@@ -0,0 +1,26 @@
+import { StyleSheet } from 'react-native';
+import { MapPropsFunction } from './format';
+
+/**
+ * Preserve props that would cause an element to not be visible while stripping
+ * the rest
+ */
+export const mapVisibilityRelatedProps: MapPropsFunction = ({
```

Comment:
`mapPropsForQueryError` or similar might be more accurate?

Reply:
Updated to use this naming.